multihost_job.py

```diff
@@ -154,6 +159,49 @@ def run_create_resources(startup_script_file):
   captured_output = subprocess.run(command, check=False, shell=True, capture_output=True)
   return captured_output
 
+def run_create_resources_curl(startup_script):
+  """ Run the Create Queued Resources (CQR) request with curl command"""
+  data = {
+        "tpu": {
+            "node_spec": [
```

Comment:
Remove the bracket on this line and the closing one on line 188. It is unecessary and removing it allows it to make it easier flexiblity